docs/proposals/2018-07-03_allowed_websites.md

```diff
@@ -0,0 +1,100 @@
+# Forbidden Websites Proposal
+
+## Problem
+
+Now that the desirable content-sharing effects of multisite have been realized, the undesirable content-sharing effects
+are also being realized. Specifically, the ability for certain websites to access and re-publish content that they do not
+have the legal right to. A common example of this is images licensed for use for a particular website but not for an organization
+at large. Multisite organizations need a way to prevent indicate that certain content should never be rendered in
+certain contexts regardless of what authors or editors attempt to do.
+
+## Proposed Solution
+
+An `forbidden.websites` property will be added to the `content_restrictions` trait.  `forbidden.websites` will contain a list of 
+website ids that may *never* display the content. Websites included on this list will be understood to be forbidden from ever seeing this content
+in a reader-facing context. A special value `"*"` will be allowed that may be used to indicate that a piece of content is
+forbidden on all websites. The absence of `forbidden.websites` should be understood to mean the same as the value `[]` -- that 
+no website is forbidden from rendering this content.
+
+## Example
+
+```
+{
+  "type": "story",
+  "version": "0.7.1",
+  
+  "headlines": {
+    "basic": "Dog Bites Man"
+  },
+  
+  "content_restrictions": {
+    "forbidden": {
+      "websites": ["the-herald"]
+    }
+  },
+  
+  "revision": {
+    "published": true
+  },
+  
+  "websites": {
+    "the-times": {
+      "website_section": {
+        "type": "section",
+        "version": "0.7.1",
+        "_id": "/news",
+        "_website": "the-times",
+        "path": "/news"
+      },
+      "website_url": "/news/dog-bites-man"
+    }
+  }
+}
+```
+
+In the above example, "Dog Bites Man" is *available* to be published to The Times and The Post. 
+It is currently published only on The Times. It may never be published on The Herald unless the content restrictions 
+are also changed.
+
+## Questions 
+
+*Who is responsible for enforcing the content restrictions?*
+
+Rendering applications (PB Templates, Apps) will ultimately be responsible for omitting restricted content.  
```

Comment:
I'm not sure if this is a good idea. We want least amount of business logic in rendering layer. Since Content API already has business logic to filter out content based on websites, shouldn't Content API own the business logic of enforcing this?